src/scripts/main.js

```diff
@@ -2,6 +2,25 @@
 
 const pushNotification = (posTop, posRight, title, description, type) => {
   // write code here
+  const newElement = document.querySelector('body');
+  const createMessage = document.createElement('div');
```

Comment:
```suggestion
  const createdMessage = document.createElement('div');
```
createMessage - создай сообщение
createdMessage - созданное сообщение